contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -18,21 +23,229 @@
 from test_cases.capture_window import Capture
 from test_cases.live_tab import VerifyFunctionCallCount
 
+Module = namedtuple("Module", ["name", "path", "is_loaded"])
+CACHE_LOCATION = "{appdata}\\OrbitProfiler\\cache\\".format(appdata=os.getenv('APPDATA'))
+
 
 def _show_symbols_and_functions_tabs(top_window):
     logging.info("Showing symbols tab")
     find_control(top_window, "TabItem", "Symbols").click_input()
 
 
-class LoadSymbols(E2ETestCase):
+def _find_and_close_error_dialog(top_window) -> str or None:
+    window = find_control(top_window, 'Window', 'Error loading symbols', raise_on_failure=False)
+    if window is not None:
+        error_message = find_control(window, 'Text').texts()[0]
+        re_result = re.search('for module "(.+?)"', error_message)
```

Comment:
I am not very familiar with python, so I assume the re means regular expression. Maybe the naming could be something like `module_search_result` rather than `re_result`. Its more about the search than it being a regular expression. (But then I also dont understand why [1] is necessary and whats in [0], so maybe im missing something).

Reply:
Renamed it to `module_path_search_result` for clarity. The [1] is due to how re.search works: Element 0 contains the entire matched string, and the following elements contain the individual groups (the parts in brackets within the pattern).